services/api-server/src/simcore_service_api_server/models/domain/projects.py

```diff
@@ -1,8 +1,10 @@
-# mypy: disable-error-code=truthy-function
-from models_library.projects import AccessRights, Node, StudyUI
-from models_library.projects_nodes import InputTypes, OutputTypes
+from models_library.projects_access import AccessRights
+from models_library.projects_nodes import InputTypes, Node, OutputTypes
 from models_library.projects_nodes_io import SimCoreFileLink
 
+# mypy: disable-error-code=truthy-function
```

Comment:
Question: why you do not put this on top?

Reply:
class issue with comments between imports ... when they are automatically rearranged it gets in the middle